### ISSUE TITLE
Unable to Reproduce Expected Results with best.pth Checkpoint on VSCrowd

### ISSUE THREAD

**Issue by pinzhihuang** [streamer-AP/CGNet#9]
Thanks for your awesome work and the provided codes! I have been following the instructions to run the best.pth checkpoint on VSCrowd, but I am not obtaining the results reported in your paper. I have double-checked my data files to ensure they follow the specified formats, yet the performance still deviates from the expected outcomes.

**Details:**
Annotation Files Format:
<image_filename> <width> <height> [<x1> <y1> <x2> <y2> <pt_x> <pt_y> <ID>][<x1> <y1> <x2> <y2> <pt_x> <pt_y> <ID>]...
<image_filename> <width> <height> [<x1> <y1> <x2> <y2> <pt_x> <pt_y> <ID>][<x1> <y1> <x2> <y2> <pt_x> <pt_y> <ID>]...
...

**Locater Data Provided:**
(x, y)
904.5, 383.3
598.7, 553.5
...

Hyperparameter:
**1. Suggested by the repo**
interval = 15, ttl = 5, max_mem=5,threshold=0.4: 
MAE:21.55, MSE:1377.57, WRAE:17.81%, RMSE:37.12
MAE by Count Ranges:
[0-49]: 5.86 
[50-99]: 8.40
[100-149]: 13.32 
[150-199]: 22.54 
[200+]: 75.22

**2. Suggested by the paper**
interval = 15, ttl = 3, max_mem=5,threshold=0.7:
MAE:99.84, MSE:17438.82, WRAE:90.01%, RMSE:132.06
MAE by Count Ranges:
[0-49: 32.21    
[50-99]: 51.38     
[100-149]: 127.65    
[150-199]: 161.31  
[200+]: 198.43 

Questions:
Data Preprocessing: Before running the model, are there any additional preprocessing steps or configurations required for the annotation and locater data?
Checkpoint: Could you verify that this checkpoint is the most updated one?
Known Issues: Are there any known issues or troubleshooting tips that might help replicate the results, as stated in the paper?

Video data: https://github.com/HopLee6/VSCrowd-Dataset/tree/main
The checkpoint was downloaded here: https://github.com/streamer-AP/CGNet/issues/4

**Comment by streamer-AP** [comment]
Hi, could you check the number of videos within the count ranges? The number seems wrong...

**Comment by streamer-AP** [comment]
The problem may come from the division of the train-test set. We follow the division of DR.VIC (https://pan.baidu.com/s/1OYBSPxgwvRMrr6UTStq7ZQ?pwd=64xm)

**Comment by svenzer007** [comment]
@streamer-AP 
Thank you for your work!

I am not able to reproduce the result.

I have used the dataset as mentioned by you - https://pan.baidu.com/s/1OYBSPxgwvRMrr6UTStq7ZQ?pwd=64xm - preprocessed with the annotation format as required: -
<image_filename> [ <pt_x> <pt_y> ][ <pt_x> <pt_y> ]
<image_filename> [ <pt_x> <pt_y> ][ <pt_x> <pt_y> ]

Moreover we used the ground truth instead of the crowd localization result, which should have only improved the results.
Following are the results obtained by me: -

********************** Repo configuration************************
Reported in repo: - MAE - 8.64, RMSE - 18.70, WRAE - 11.76
My results: -  MAE:16.93, RMSE:23.51, WRAE:27.34%
*******************************************************************

************************ Paper configuration*********************
Reported in paper: - MAE: 8.86, RMSE:17.69, WRAE:12.6%
My results: - MAE:358.20, RMSE:498.91, WRAE:470.15%
*******************************************************************

**Comment by streamer-AP** [comment]
MAE =358 is very abnormal result.. Could you check the data you generate or test with the provided localization result.

**Comment by svenzer007** [comment]
Thank you for your prompt response!

I used the data - https://github.com/HopLee6/VSCrowd-Dataset/tree/main  and preprocessed them into the specified format.
And instead of the crowd localizationr results, if we use the ground truth from the data, it should only improve the results.

**Comment by streamer-AP** [comment]
It is true gt would improve the result..
Could you check your data or upload it then I can help you check it.

**Comment by svenzer007** [comment]
Please find the test data I used to perform inference - https://drive.google.com/drive/folders/11Qk-_AlYAiuzvpdJ6cORdx89s73vWZlD?usp=sharing
